server/models/user.js

```diff
@@ -1,11 +1,40 @@
 const mongoose = require('mongoose');
-
-const Schema = mongoose.Schema;
+const { Schema } = mongoose;
 
 const userSchema = new Schema({
-  // To Do: define user using mongoose.js
+  first_name: { type: String, required: true },
+  last_name: String,
+  password: { type: Number, required: true },
+  // Do we need reset ability yet?
+  resetToke: Number,
+  resetTokenExpiry: Date,
```

Comment:
I think we can leave this out for now

Reply:
No problem!

Reply:
I'm nit picking here, but the combination of snake_case `first_name` and camelCase `resetToke` bug me

Reply:
Agree!! Fixed to be all camelCase